### PR TITLE
test: Tidy up flags test

### PR DESCRIPTION
**Pull request by phlptt**
* Seed the flag via API so that it is more reliable and quicker
* Update to latest forms and content submodules
* fix because of new content
* Tidy up of users tests

https://jenkins.ci.uktrade.io/job/lite-e2e-internal-frontend/1125/

blue build here

### CODE REVIEW

File: ui_automation_tests/step_defs/test_assign_case_flags_to_case.py [hunk 1]
```diff
@@ -14,6 +14,12 @@ def click_edit_case_flags_link(driver):
     application_page.click_edit_case_flags()
 
 
+@when("I click edit destination flags link")
+def click_edit_case_flags_link(driver):
+    application_page = ApplicationPage(driver)
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/step_defs/test_assign_case_flags_to_case.py]: This could be oneline

File: ui_automation_tests/step_defs/test_users.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+@then("I see new user")
+def see_new_user(driver, context):
+    assert utils.paginated_item_exists(context.added_email, driver), "Item couldn't be found"
+
+
+@when("I deactivate new user")
+def deactivate_user(driver, context):
+    user_page = UsersPage(driver)
+    user_page.go_to_users_page(context)
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/step_defs/test_users.py]: Why do you use user_page to go to user page (seems odd)

File: ui_automation_tests/step_defs/test_assign_case_flags_to_case.py (hunk 1 above)
Comment (phlptt) [reply ui_automation_tests/step_defs/test_assign_case_flags_to_case.py]: done :)

File: ui_automation_tests/step_defs/test_users.py (hunk 2 above)
Comment (phlptt) [reply ui_automation_tests/step_defs/test_users.py]: all of this has just been moved but i'll change it

Comment (phlptt) [reply ui_automation_tests/step_defs/test_users.py]: actually it does make sense ists users page so the users list

**Pull request merged**